charts/gitness/values.yaml

```diff
@@ -0,0 +1,107 @@
+# Default values for gitness.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: harness/gitness
+  pullPolicy: IfNotPresent
```

Comment:
PullPolicy are advised be 'Always' if you are using latest tag. Please add a comment for users as well

Reply:
> 如果您使用最新标签，建议 PullPolicy 设置为“始终”。还请用户添加评论

we have offline environment,  maybe should not `always` policy.

And maybe should with fixed labels and clear versions, this is the standard for production environments.

Reply:
Done :)